Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs

```diff
@@ -231,19 +335,19 @@ protected override void EndProcessing()
                     progressIds.Add(progressId);
                 }
 
-                if (ParameterSetName == "ParameterGroup" && !StartAfter.HasValue)
+                if (!StartAfter.HasValue)
                 {
-                    // Create BackgroundTask 
-                    WriteDebug($"Create BackgroundTask EventType[{EventType}] RawInputData.length[{RawInputData.Length}]");
+                    // Create BackgroundTask
+                    WriteDebug($"Create BackgroundTask EventType[{EventType}] RawInputData.length[{inputData}] StartAfter[{StartAfter}]");
```

Comment:
Why print StartAfter value, when two lines up you checked it does not have a value... please do similar correction as higher.
Orginal was `WriteDebug($"Create BackgroundTask EventType[{EventType}] RawInputData.length[{RawInputData.Length}]");`